lib/instruction-parser.ts

```diff
@@ -0,0 +1,45 @@
+export { getPackagesFromRunInstructions, DockerFilePackages };
+
+interface DockerFilePackages {
+  [packageName: string]: {
+    instruction: string;
+  };
+}
+
+// tslint:disable-next-line:max-line-length
+const installRegex = /\s*(rpm\s+-i|rpm\s+--install|apk\s+add|apt\s+install|apt-get\s+install)\s+(-\w\s+)?/;
```

Comment:
Maybe we should also support more common ones? like `yum install` and `aptitude install`?

Reply:
Done